test/robot/go_robot_go.sh

```diff
@@ -0,0 +1,320 @@
+#!/bin/bash
```

Comment:
add copyright header and a short summary of what this does.

Reply:
done.